tests/AddressBookGUITest.java

```diff
@@ -32,19 +32,35 @@
 import org.junit.jupiter.api.Test;
 import org.junit.rules.TemporaryFolder;
 
+/**
+ * AddressBookGUITest.java
+ * This is the testcase for testing the GUI itself.
+ * It uses AssertJ to perform automated tests
+ *
+ *
+ */
+
 //NOTE: Due to a bug(s) in AssertJ/JDK, this entire class's
 //tests fail on MacOS systems. This is partly because of the way
 //the system simulates clicks on the GUI. It requires special
 //security permissions to access the mouse and keyboard.
 //SEE: https://github.com/joel-costigliola/assertj-swing/issues/25
 public class AddressBookGUITest {
 
+
+    //TemporaryFolder clears itself from memory whne program ends
     @Rule
     public static TemporaryFolder folder = new TemporaryFolder();
     private static File testFile = null;
     private static FrameFixture window = null;
     private static AddressBookGUI addressBookGUI = null;
 
+    /**
+     * This class runs before each test, to initialize variables.
```

Comment:
This "method" not class runs before each

Reply:
fixed